app/examples/react-18/src/components/TimeSeriesStaticTest.jsx

```diff
@@ -81,7 +68,11 @@ export function TimeSeriesStaticTest() {
         styles={{
           bar: { backgroundColor: barsColor },
           canvas: { backgroundColor: '#f5f5f5' },
-          point: { style: pointStyle }
+          point: { style: pointStyle },
+          yAxis: {
+            beginAtZero: true,
+            type: 'logarithmic'
```

Comment:
```suggestion
            scale: 'logarithmic'
```

Let's call the prop `scale` not `type`. It is more specific

Reply:
This is done